src/modules/Dashboard/components/ParticipantTagChip/ParticipantTagChip.types.ts

```diff
@@ -0,0 +1,6 @@
+import { ParticipantTag } from 'shared/consts';
+
+export type ParticipantTagProps = {
```

Comment:
Totally, thanks for noticing… It was a relic from when the component was still called `ParticipantTag`, but that identifier conflicted with the enum `ParticipantTag` and I wanted to avoid confusion.